modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -1,339 +1,425 @@
 
+Runtime Fabric contains all of the components it requires. These components, including Docker and Kubernetes, are optimized to work efficiently with Mule runtimes and other MuleSoft services.
 
-For applications that are deployed elsewhere, you must carry out these Runtime updates manually.
+To use the Runtime Fabric option, you first create a Runtime Fabric using Runtime Manager.
+Then, you install Runtime Fabric on your infrastructure.
+Finally, you deploy your applications from your on-premises Runtime Manager to either a server, server group, or cluster.
 
+The following features and limitations apply when using Runtime Manager to manage Runtime Fabric deployments:
 
-== Other Components
 
-There are some components for which CloudHub has limited support for currently:
+* *High availability*
++
+Runtime Fabric provides automatic application fail-over.
 
-* Distributed locks: currently, CloudHub cannot coordinate invocations of FTP and File endpoints across multiple workers.
-* Idempotent routers works with in memory stores and according to the limitations of the CloudHub Object Store if you configure it to use it. If those options do not fit your needs, you can use another Object Store.
+* *Load balancing*
++
+Runtime Fabric does not include the load balancing feature. 
+You can manage load balancing with the tools connected to your infrastructure.
 
+* *Logging*
++
+Runtime Fabric uses Ops Center to view and configure logs. 
+You can use forward logs from Runtime Fabric to a rsyslog server.
++
+See xref:runtime-fabric::runtime-fabric-logs.adoc[View and Configure Logging in Runtime Fabric].
 
-== Deployment Strategy Flexibility
+* *Monitoring*
++
+Runtime Fabric provides dashboards and alerts in Ops Center for viewing performance and availability metrics. To receive and send alerts, use an SMTP server.
++
+See https://docs.mulesoft.com/runtime-fabric/1.4/configure-alerting.adoc[Configure Alerting on Runtime Fabric].
++
+You can also use Anypoint Monitoring to view application metrics with Anypoint Platform.
++
+See xref:runtime-fabric::manage-monitor-applications.adoc[Monitoring Applications Deployed to Runtime Fabric].
 
-If you want to deploy the same Mule application via various different deployment strategies, such as to an <<Cloud Console On-Premises Deployment, on-premises server>> and <<Cloud Console CloudHub Deployment, CloudHub>>, abstract some parameters of the application to xref:mule-runtime::intro-packaging.adoc[application properties] that you can set with different values in each use case, without needing to alter the actual application.
+* *Object store*
++
+Runtime Fabric does not support Object Store.
```

Comment:
We should put the same message here as we do for Hybrid deployments -- customers must set up their own database.

We can also mention that Mule clustering is supported between replicas of an application.

Reply:
@sundippatel - should your second suggestion go under HA rather than OS?